### DISCUSSION TITLE
予約投稿

### DISCUSSION THREAD

**Discussion by hanaseleb** [increments/qiita-discussions#600] [要望・アイデア・不具合]
投稿日時を指定できる予約投稿機能が欲しい。夜中に書き上げてしまった記事は翌朝に投稿したいですよねー。

土日に書いた記事も、場合によっては月曜日にあげたいなってときもありますし。

**Comment by tanaka-asami415** [comment]
@hanaseleb 
リクエストありがとうございます🙏
有料プラン等でのご提供になるかもしれませんが、将来的に検討したいと思っています！
